### PR TITLE
[Fix] Fixed the modal screen fit issue

### LINKED ISSUE #1803: [Catalog] Modals on Meshery Catalog are not fitting to the screen
Add or update the following [Meshery Catalog](https://meshery.io/catalog) item(s):

**Name:** Modal Screen Fit Issue  
**Type:** UI Bug  
**Compatibility:** All Browsers  
**Pattern ID:** N/A  
**image:** 
Screenshot
**Pattern Info:** Modals on the Meshery Catalog are not fitting to the screen. They are exceeding the screen boundaries, making the content not completely visible.  
**Pattern Caveats:** N/A  
**URL:** [Meshery Catalog](https://meshery.io/catalog)  
**Download link:** 

**Steps to Reproduce:**
1. Go to [Meshery Catalog](https://meshery.io/catalog).
2. Click on any item to open a modal.
3. Observe that the modal content exceeds the screen boundaries and is not fully visible.

**Expected Behavior:**
The modals should be fully visible and fit within the screen boundaries, ensuring that all content is accessible and readable.

**Environment:**
- Browser: Chrome
- Operating System: Windows

### Contributor Resources

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](https://slack.meshery.io) for access.

### PR DESCRIPTION
**Pull request by kanishk6103**
**Description**

This PR fixes #1803

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@amitamrutiya  would you like to offer review on this PR?

File: _sass/catalog.scss [hunk 1]
```diff
@@ -1020,6 +1020,18 @@ label {
   background: #00b39f;
 }
 
+@media screen and (max-width: 550px) {
+  .btn-grp{
+    flex-direction: row;
+    flex-wrap: wrap;
+    gap: 1rem;
+    // padding-bottom: 2rem;
```
Comment (amitamrutiya) [inline _sass/catalog.scss:1028]: Remove this comment

**Review by amitamrutiya — COMMENTED** [review]
Remove the extra comment; otherwise, LGTM.

**Comment by Ashparshp** [comment]
@kanishk6103, Thanks for your contribution, let's discuss this on the website's call. Please add this as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk).

**Review by amitamrutiya — APPROVED** [review]

**Pull request closed without merging**